test/steps/maven.go

```diff
@@ -22,6 +22,7 @@ import (
 // registerMavenSteps register all existing Maven steps
 func registerMavenSteps(s *godog.Suite, data *Data) {
 	s.Step(`^Local example service "([^"]*)" is built by Maven$`, data.localServiceBuiltByMaven)
+	s.Step(`^Local example service "([^"]*)" is built by Maven using "([^"]*)"$`, data.localServiceBuiltByMavenWithProfile)
```

Comment:
```suggestion
	s.Step(`^Local example service "([^"]*)" is built by Maven using profile "([^"]*)"$`, data.localServiceBuiltByMavenWithProfile)
```
one last thing I did not noticed. I think it is cleared and it will be understandable when looking at the logs

Reply:
added